src/main/kotlin/app/ui/AudioPlayer.kt

```diff
@@ -0,0 +1,10 @@
+import javafx.scene.media.AudioClip
+
+fun playKotlin(URI: String) {
+    val myClip = AudioClip(URI)
+    myClip.play()
+    while(myClip.isPlaying) {
+        Thread.yield()
+    }
+    println("done playing")
```

Comment:
We may want to introduce a logging class where things like this are done, but we probably don't want to be always writing things like this to standard out